bids/bids.tf

```diff
@@ -0,0 +1,306 @@
+      "class" : "baremetal_2a5"
+    },
+    "0a768cab-6e80-4012-ac79-91555ee9bad4" : {
+      "facility" : "e1e9c52e-a0bc-4117-b996-0fc94843ea09",
+      "class" : "baremetal_2a5"
+    },
+    "86a5c1f1-453a-4719-bd93-8427272c4073" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "13fea8e9-9184-42fb-b464-f8951897ff68" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "68ead1e9-a310-4744-b90b-d06d5d31afc3" : {
+      "facility" : "e1e9c52e-a0bc-4117-b996-0fc94843ea09",
+      "class" : "baremetal_2a5"
+    },
+    "e8977920-13c6-484d-b4f4-92426583cad0" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "de28365b-1bcd-492b-9f91-f775327f7425" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "6b25844f-aaa8-4e50-94a3-6b175077b90c" : {
+      "facility" : "e1e9c52e-a0bc-4117-b996-0fc94843ea09",
+      "class" : "baremetal_2a5"
+    },
+    "533a5afa-a56e-4766-ab7e-fdc236d24449" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "3f4138c8-828d-46fb-84bf-762810092697" : {
+      "facility" : "e1e9c52e-a0bc-4117-b996-0fc94843ea09",
+      "class" : "c2.large.arm"
+    },
+    "aa3fee8e-3cdf-4562-af26-c36cff180827" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "0f22764a-707d-4279-acfe-862bd6f2607d" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "6f21cbbd-35d9-4426-895b-8c8e218e339a" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "fbbc4c82-dad8-4ff0-a5e4-f028910ac2ce" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "89873ca4-8345-4529-9762-669df5fb175e" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "43e2dbd2-451e-485e-bd98-206d44af4b8c" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    },
+    "218a7f82-24e8-4b00-8891-a32510b0d32b" : {
+      "facility" : "c9dcbd06-6797-4096-b648-1be16dd5d833",
+      "class" : "baremetal_2a5"
+    }
+  }
+}
+
+variable "reservation_class_urls" {
+  type = map(string)
+  default = {
+    "baremetal_2a5" = "https://netboot.gsc.io/hydra-aarch64-linux/netboot.ipxe"
+    "c2.large.arm"  = "https://netboot.gsc.io/hydra-aarch64-linux/netboot.ipxe"
+  }
+}
+
+variable "reservation_facility_names" {
+  type = map(string)
+  default = {
+    "c9dcbd06-6797-4096-b648-1be16dd5d833" = "dfw2",
+    "e1e9c52e-a0bc-4117-b996-0fc94843ea09" = "ewr1",
+    "d0c0d80d-4637-4fa0-b79e-803fc42b19bd" = "mrs1"
+  }
+}
+
+variable "reservation_broken" {
```

Comment:
Hm, maybe add a description here. What is this? Some kind of ignore list?

Reply:
All set!